lib/transcoder/audio.ex

```diff
@@ -34,7 +48,11 @@ defmodule Boombox.Transcoder.Audio do
   end
 
   defp do_plug_audio_transcoding(builder, %RemoteStream{content_format: Opus}, %Opus{}) do
-    builder |> child(:opus_parser, Opus.Parser)
+    builder
```

Comment:
You cannot remove plugging `Opus.Parser`, the output stream format will remain `%RemoteStream{content_format: Opus}` but it should be `%Opus{}`